Code/GraphMol/FilterCatalog/Wrap/FilterCatalog.cpp

```diff
@@ -348,7 +348,7 @@ struct filtercat_wrapper {
         .def("GetName", &FilterMatcherBase::getName, python::args("self"))
         .def("__str__", &FilterMatcherBase::getName, python::args("self"));
 
-    python::register_ptr_to_python<boost::shared_ptr<FilterMatcherBase>>();
+    //python::register_ptr_to_python<boost::shared_ptr<FilterMatcherBase>>();
```

Comment:
I think you probably meant to just delete this line. :-)

Reply:
I did in the version I didn't push :)